src/components/Footer.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+function Footer() {
+  const HotelContact = [
```

Comment:
By convention, `const` names start with lower case letters in javascript, so this would be `hotelContact`.

React function components are an exception to this general rule.